.github/workflows/tests-auth.yml

```
@@ -1,12 +1,13 @@
+      - "auth/**"
 
 jobs:
   build:
-     runs-on: ubuntu-latest
-     steps:
-       - uses: actions/checkout@v2
-       - run: cd auth && npm install && npm run test:ci
-  
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)